src/main/java/dev/pgm/community/commands/VanishedCommand.java

```diff
@@ -0,0 +1,47 @@
+package dev.pgm.community.commands;
+
+import static net.kyori.adventure.text.Component.text;
+
+import dev.pgm.community.CommunityCommand;
+import dev.pgm.community.CommunityPermissions;
+import dev.pgm.community.utils.CommandAudience;
+import java.util.List;
+import java.util.stream.Collectors;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.Bukkit;
+import tc.oc.pgm.lib.cloud.commandframework.annotations.CommandDescription;
+import tc.oc.pgm.lib.cloud.commandframework.annotations.CommandMethod;
+import tc.oc.pgm.lib.cloud.commandframework.annotations.CommandPermission;
+import tc.oc.pgm.util.named.NameStyle;
+import tc.oc.pgm.util.player.PlayerComponent;
+import tc.oc.pgm.util.text.TextFormatter;
+
+public class VanishedCommand extends CommunityCommand {
+  @CommandMethod("vanished|lv|listv")
+  @CommandDescription("View a list of online vanished players")
+  @CommandPermission(CommunityPermissions.VIEW_VANISHED)
+  public void viewVanished(CommandAudience viewer) {
+    List<Component> vanishedNames =
+        Bukkit.getOnlinePlayers().stream()
+            .filter(player -> player.hasMetadata("isVanished"))
```

Comment:
Also probably better to use
```suggestion
            .filter(Integration::isVanished)
```